Detectors/gconfig/g3libs.C

```diff
@@ -20,11 +20,11 @@ void g3libs()
 {
   cout << "Loading Geant3 libraries ..." << endl;
 
-  if (isLibrary("libdummies.so"))
+  if (isLibrary("libdummies"))
      gSystem->Load("libdummies.so");
```

Comment:
what does "dummies" mean? should there be a better name or some comment what you are doing there?